Extensions/PrimitiveDrawing/shapepainterruntimeobject-pixi-renderer.ts

```diff
@@ -485,8 +486,35 @@ namespace gdjs {
       point[1] = position.y;
       return point;
     }
+    applyAntialiasing(): void {
```

Comment:
```suggestion
    
    updateAntialiasing(): void {
```

For consistency with other function names